src/accounts/userservice/db.py

```diff
@@ -70,7 +70,7 @@ def generate_accountid(self):
         accountid = None
         with self.engine.connect() as conn:
             while accountid is None:
-                accountid = str(random.randint(1e9, (1e10 - 1)))
+                accountid = str(random.randint(1_000_000_000, (10_000_000_000 - 1)))
```

Comment:
Today I learned you can define numbers like `10_000` (using underscores) in Python. Very cool feature.

Reply:
Yep! Technically not required (I could've just put `1000000000`) but it helps with readability.

The fix wasn't the underscore, but rather going from `1e9` (which is a `float`) to an `int`.